<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adjpara</class>
 <widget class="QDialog" name="adjpara">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>数据关联参数选择</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>107</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>卡尔曼滤波参数选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>301</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>事故判定参数选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>550</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>550</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>690</y>
     <width>5502</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>50</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>计算iou阈值(iou_threshold)：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_datass">
      <property name="placeholderText">
       <string>请输入一个0～100的数字</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>150</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>kf_.noise_rx:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_kalman1">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>kf_.noise_ax:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_kalman5">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>181</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,1">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>kf_.noise_ry:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_kalman2">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>kf_.noise_ay:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_kalman6">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>212</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1,1">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>kf_.noise_rw:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_kalman3">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>kf_.noise_aw:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_kalman7">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>243</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,1,1">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>kf_.noise_rh:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_kalman4">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>kf_.noise_ah:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_kalman8">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>340</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>移动因数(kmove):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge1">
      <property name="placeholderText">
       <string>请输入一个0～0.5的小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>371</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>疑似对象检查帧数(like_check_thre):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge2">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个整数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>402</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>未出事故消除对象检查帧数(noacci_count_thre):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge3">
      <property name="placeholderText">
       <string>请输入一个整数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>433</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>事故对象相撞对象检查帧数(neigh_check_thre):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge4">
      <property name="placeholderText">
       <string>请输入一个整数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>464</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>判定为事故速度阈值(vfilter_thre):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge5">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>495</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>低速下限(vfilter_lowthre):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge6">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>526</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>y向行驶相撞物体的vyf下限(vy_filter_thre:)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge7">
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>557</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>vy_filter的放大增益(vyf_gain):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge8">
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>588</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>认为静止的速度阈值(vsum_staythre):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge9">
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>619</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>认为静止的坐标波动阈值(slide_staythre):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge10">
      <property name="placeholderText">
       <string>请输入一个小数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>650</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="11,6">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>检查相撞物体是否静止的帧数(stay_checkthre:)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_judge11">
      <property name="placeholderText">
       <string>请输入一个整数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>711</y>
     <width>291</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,2,1">
    <item>
     <widget class="QPushButton" name="btn_ensure">
      <property name="text">
       <string>确定</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="text">
       <string>取消</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
